code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -103,24 +104,37 @@
 		use_autosurgeon()
 	return TRUE
 
-/obj/item/autosurgeon/organ/cmo
+/obj/item/autosurgeon/medical_hud
+	name = "autosurgeon"
 	desc = "A single use autosurgeon that contains a medical heads-up display augment. A screwdriver can be used to remove it, but implants can't be placed back in."
 	uses = 1
 	starting_organ = /obj/item/organ/internal/cyberimp/eyes/hud/medical
 
-/obj/item/autosurgeon/organ/syndicate/laser_arm
+
+/obj/item/autosurgeon/syndicate
+	name = "suspicious autosurgeon"
+	icon_state = "autosurgeon_syndicate"
+	surgery_speed = 0.75
+	loaded_overlay = "autosurgeon_syndicate_loaded_overlay"
+
+/obj/item/autosurgeon/syndicate/laser_arm
 	desc = "A single use autosurgeon that contains a combat arms-up laser augment. A screwdriver can be used to remove it, but implants can't be placed back in."
 	uses = 1
 	starting_organ = /obj/item/organ/internal/cyberimp/arm/gun/laser
 
-/obj/item/autosurgeon/organ/syndicate/thermal_eyes
+/obj/item/autosurgeon/syndicate/thermal_eyes
 	starting_organ = /obj/item/organ/internal/eyes/robotic/thermals
 
-/obj/item/autosurgeon/organ/syndicate/xray_eyes
+/obj/item/autosurgeon/syndicate/xray_eyes
 	starting_organ = /obj/item/organ/internal/eyes/robotic/xray
 
-/obj/item/autosurgeon/organ/syndicate/anti_stun
+/obj/item/autosurgeon/syndicate/anti_stun
 	starting_organ = /obj/item/organ/internal/cyberimp/brain/anti_stun
 
-/obj/item/autosurgeon/organ/syndicate/reviver
+/obj/item/autosurgeon/syndicate/reviver
 	starting_organ = /obj/item/organ/internal/cyberimp/chest/reviver
+
+/obj/item/autosurgeon/syndicate/commsagent
+	desc = "A device that automatically - painfully - inserts an implant. It seems someone's specially \
+	modified this one to only insert... tongues. Horrifying."
```

Comment:
Looks like this can use any type of organ now? I wanted to avoid doing that as it'd be a massive buff to anyone who raided the lavaland syndies (or now the space comms agent too, apparently)

Reply:
It looks like the tongue organ whitelist never worked, that was just fluff. I added it in now though. The old comms agent implanter didn't even work in the first place, I think.